.github/workflows/build_test_push.yaml

```diff
@@ -62,8 +62,21 @@ jobs:
             GIT_PERSONAL_ACCESS=${{ secrets.GIT_PERSONAL_ACCESS }}
             GIT_USER_ID=${{ secrets.GIT_USER_ID }}
 
-      - name: run unit test
-        run:  docker run us.gcr.io/model-159019/gh:${{ github.sha }}-dev /bin/bash -c "mvn test"
+      - name: Download + setup micro norcal test files
+        run: |
+          export MININORCAL_OSM_PATH=${{ secrets.MININORCAL_OSM_PATH}}
+          export MININORCAL_GTFS_PATH=${{ secrets.MININORCAL_GTFS_PATH}}
+          ./setup_tests.sh
+
+      # see https://github.com/docker/compose/issues/5696 for info on COMPOSE_INTERACTIVE_NO_CLI env var
+      - name: run unit tests
+        run: |
+          docker run -t -d --name testrunner us.gcr.io/model-159019/gh:${{ github.sha }}-dev /bin/bash
+          docker exec -i testrunner /bin/bash -c "mkdir -p ./web/test-data/gtfs"
```

Comment:
note here: for some reason, running `exec` with `-it` was complaining at me here; removing the `-t` made things work. Not an issue in circle, so didn't change anything there 🤷